packages/inference/package.json

```diff
@@ -4,10 +4,7 @@
 	"license": "MIT",
 	"author": "Tim Mikeladze <tim.mikeladze@gmail.com>",
 	"description": "Typescript wrapper for the Hugging Face Inference API",
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/huggingface/huggingface.js.git"
-	},
+	"repository": "https://github.com/huggingface/huggingface.js.git",
```

Comment:
Let's not change unnecessarily

Reply:
Ops I must have changed this by accident while updating the hub one. First time I changed all of them was when I was testing publishing to npm so I needed to change repo to the fork under my account so I can publish. Since I'm used to the `github:user/repo-name` format, I automatically use it when filling out this field. Anyways I changed it back.